### PR TITLE
Jobim2020

### PR DESCRIPTION
**Pull request by MaxUlysse**
- remove registration links from hackathon 2019 in Stockholm, and hackathon 2020 in London
- add slides to presentation at JOBIM2020

### CODE REVIEW

File: markdown/events/2020/jobim-2020.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ Maxime Garcia will be giving a 20-minute talk about the nf-core community.
 The talk will be available for anyone to watch after the conference.
 It will be posted on this page when ready.
 
+<div class="row">
+    <div class="col-md-6">
+        <iframe src="https://maxulysse.github.io/jobim2020" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
+        <div style="margin-bottom:5px"> <strong> <a href="https://maxulysse.github.io/jobim2020" title="nf-core, a community effort for collaborative, peer-reviewed analysis pipelines" target="_blank">nf-core, a community effort for collaborative, peer-reviewed analysis pipelines</a> </strong> from <strong><a href="https://maxulysse.github.io" target="_blank">Maxime Garcia</a></strong> </div>
+    </div>
+</div>
```
Comment (ewels) [inline markdown/events/2020/jobim-2020.md:32]: Nice! Two minor suggestions:

* I don't think that we need the `col-md-6` to limit it to half of the width of the screen? There's nothing on the right so it just gives a bunch of whitespace
* The aspect ratio felt a little odd to me. Upping the width to `700` feels better

```suggestion
<iframe src="https://maxulysse.github.io/jobim2020" width="700" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe>
<div style="margin-bottom:5px"> <strong> <a href="https://maxulysse.github.io/jobim2020" title="nf-core, a community effort for collaborative, peer-reviewed analysis pipelines" target="_blank">nf-core, a community effort for collaborative, peer-reviewed analysis pipelines</a> </strong> by <strong><a href="https://maxulysse.github.io" target="_blank">Maxime Garcia</a></strong> </div>
```

image

File: markdown/events/2020/jobim-2020.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ Maxime Garcia will be giving a 20-minute talk about the nf-core community.
 The talk will be available for anyone to watch after the conference.
 It will be posted on this page when ready.
```
Comment (ewels) [inline markdown/events/2020/jobim-2020.md:25]:
```suggestion
The talk will be available for anyone to watch after the conference.
You can see the slides below.
```

**Comment by MaxUlysse** [comment]
Thanks for making the modifications, I haven't touched my computer yet, and I'm just looking at my mails now...
I really does feel like vacations!!!

**Comment by ewels** [comment]
No worries! I figured that was the case 😉 Enjoy! 🏖️

**Pull request merged**